api.go

```diff
@@ -609,11 +609,25 @@ func (api *API) Import(ctx context.Context, req *ImportRequest) error {
 		return errors.Wrap(err, "validating api method")
 	}
 
-	_, field, err := api.indexField(req.Index, req.Field, req.Shard)
+	index, field, err := api.indexField(req.Index, req.Field, req.Shard)
 	if err != nil {
 		return errors.Wrap(err, "getting field")
 	}
 
+	// Translate row keys.
+	if field.keys() {
```

Comment:
I think we should validate in this case that the user hasn't passed row IDs, and similarly in the next block that the user hasn't passed any column IDs.

Reply:
Good point. I added validation in 7a5b7c3f.